### PR TITLE
Added preview pane hide/show

### PR DESCRIPTION
**Pull request by GauravV-02**
@humphd

### CODE REVIEW

**Comment by flukeout** [comment]
OK—tested, and this works great when using the following console commands, as described in #889...

* `bramble.hidePreview()`
* `bramble.showPreview()`

**Comment by flukeout** [comment]
@GauravV-02 What is the relationship of this PR to #889—should the functionality be all in one PR?

**Comment by GauravV-02** [comment]
@flukeout #889 was WIP and I pushed it to clear my doubts with humphd, this one is Final I removed the unnecessary code.

File: src/extensions/default/bramble/lib/RemoteCommandHandler.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ define(function (require, exports, module) {
     var Commands           = brackets.getModule("command/Commands");
     var FileSystem         = brackets.getModule("filesystem/FileSystem");
     var HTMLRewriter       = brackets.getModule("filesystem/impls/filer/lib/HTMLRewriter");
-    var SidebarView        = brackets.getModule("project/SidebarView");
+    var SidebarView        = brackets.getModule("project/SidebarView");
```
Comment (gideonthomas) [inline src/extensions/default/bramble/lib/RemoteCommandHandler.js]: undo this change

File: src/extensions/default/bramble/lib/RemoteCommandHandler.js [hunk 2]
```diff
@@ -25,6 +25,7 @@ define(function (require, exports, module) {
     var Tutorial = require("lib/Tutorial");
     var Theme = require("lib/Theme");
     var UI = require("lib/UI");
+    var PreviewView = require("lib/iframe-browser");
```
Comment (gideonthomas) [inline src/extensions/default/bramble/lib/RemoteCommandHandler.js]: let's follow the convention brackets has established and use the name of the module as the variable name, i.e. `IframeBrowser`

File: src/extensions/default/bramble/lib/iframe-browser.js [hunk 3]
```diff
@@ -180,46 +147,29 @@ define(function (require, exports, module) {
     /**
      * Used to show second pane, change lilveDevButton background and close the detached preview
      */
-    function attachPreview() {
```
Comment (gideonthomas) [inline src/extensions/default/bramble/lib/iframe-browser.js]: @humphd did we stop supporting this feature? I can't seem to find this being called anywhere

**Comment by humphd** [comment]
@GauravV-02 any update here?

File: src/extensions/default/bramble/lib/RemoteCommandHandler.js (hunk 1 above)
Comment (GauravV-02) [reply src/extensions/default/bramble/lib/RemoteCommandHandler.js]: Both lines are exact same, not sure what to undo.

Comment (humphd) [reply src/extensions/default/bramble/lib/RemoteCommandHandler.js]: See the extra green at the end of the line?  That's whitespace that you've added without actually changing the content of the file.  People try to avoid this in code because it ends up obscuring the history of this file/line in git--it looks like you changed it, but actually, it wasn't really changed in this patch substantively, just a cosmetic thing.

File: src/extensions/default/bramble/lib/RemoteCommandHandler.js (hunk 2 above)
Comment (GauravV-02) [reply src/extensions/default/bramble/lib/RemoteCommandHandler.js]: Did this change locally

File: src/extensions/default/bramble/lib/iframe-browser.js (hunk 3 above)
Comment (GauravV-02) [reply src/extensions/default/bramble/lib/iframe-browser.js]: Do I have to clarify something here?

**Comment by GauravV-02** [comment]
@humphd From suggested changes I did the second one, not sure about first and last

File: src/extensions/default/bramble/lib/RemoteCommandHandler.js (hunk 2 above)
Comment (humphd) [reply src/extensions/default/bramble/lib/RemoteCommandHandler.js]: Can you push these changes up here too please?

File: src/extensions/default/bramble/lib/iframe-browser.js (hunk 3 above)
Comment (humphd) [reply src/extensions/default/bramble/lib/iframe-browser.js]: Yeah, we are not using this feature any more.

**Comment by gideonthomas** [comment]
Thanks @GauravV-02, this looks good to me now. Will leave it to @humphd to merge if he's happy with it.

File: src/extensions/default/bramble/lib/iframe-browser.js [hunk 4]
```diff
@@ -32,7 +32,7 @@ define(function (require, exports, module) {
 
         // If iframe does not exist, then show it
         if(result.rows === 1 && result.columns === 1) {
-            show(_orientation);
+            setLayout(_orientation);
```
Comment (humphd) [inline src/extensions/default/bramble/lib/iframe-browser.js]: Here you're assuming that `setLayout` accepts an `orientation` argument, but below in line 71, you don't use it.  We need to sort that out.

Comment (gideonthomas) [reply src/extensions/default/bramble/lib/iframe-browser.js]: good catch...looks like a bug that Brackets had initially. I think just removing the argument passed in here should be good enough since `_orientation` is global to this module.

**Comment by GauravV-02** [comment]
@humphd Updated readme
